app/mailers/provider_mailer.rb

```diff
@@ -48,4 +48,27 @@ def application_rejected_by_default(provider_user, application_choice)
               to: provider_user.email_address,
               subject: t('provider_application_rejected_by_default.email.subject', candidate_name: @application.candidate_name))
   end
+
+  def chase_provider_decision(provider_user, application_choice)
+    @application =
+      Struct.new(
+        :candidate_name,
+        :provider_user_name,
+        :course_name_and_code,
+        :submitted_at,
+        :application_choice,
+        :rbd_date,
+      ).new(
```

Comment:
Maybe extract the definition of the struct into a constant, e.g.

```
ChaseProviderDecisionApplication = Struct.new(
        :candidate_name,
        :provider_user_name,
        :course_name_and_code,
        :submitted_at,
        :application_choice,
        :rbd_date,
      )
```

(you can almost certainly think of a better name for it!)